examples/common/utilities.adb

```diff
@@ -27,6 +34,46 @@ package body Utilities is
 
     --  ------------------------------------------------------------------------
 
+    procedure Print_GL_Array3 (Name : String; anArray : GL.Types.Single_Array) is
```

Comment:
I'd like to use GL.Objects.Buffers.Load_To_Buffer with GL.Types.Singles.Vector3_Array.
If this is possible, how to do it?
So far, I haven't been able to figure it out.